endpoints/openrtb2/sample-requests/invalid-stored/bad_incoming_1.json

```diff
@@ -1,41 +1,42 @@
 {
-    "description": "Otherwise valid request using stored request; incoming request has a comma after the closing curly brace of the second set of dimensions to yield invalid JSON",
-  
-    "message": "Invalid request: Invalid JSON in Incoming Request: invalid character ']' looking for beginning of value at offset 377\n",
-    "requestPayload": {
-      "id": "some-request-id",
-      "site": {
-        "page": "prebid.org"
-      },
-      "imp": [
-        {
-          "id": "some-impression-id",
-          "banner": {
-            "format": [
-              {
-                "w": 300,
-                "h": 250
-              },
-              {
-                "w": 300,
-                "h": 600
-              },
-            ]
-          },
-          "ext": {
-            "appnexus": {
-              "placementId": 12883451
-            }
+  "description": "Otherwise valid request using stored request; incoming request has a comma after the closing curly brace of the second set of dimensions to yield invalid JSON",
+
+  "mockBidRequest": {
+    "id": "some-request-id",
+    "site": {
+      "page": "prebid.org"
```

Comment:
I believe this file shouldn't be moved because files in this directories seem give coverage to different parts of the code. For instance, this test covers an error scenario in the `processStoredRequests` funciton:
```
1072 func (deps *endpointDeps) processStoredRequests(ctx context.Context, requestJson []byte) ([]byte, []error) {
1073 +-- 22 lines: Parse the Stored Request IDs from the BidRequest and Imps.--------------------------
1095     if hasStoredBidRequest {
1096         resolvedRequest, err = jsonpatch.MergePatch(storedRequests[storedBidRequestId], requestJson)
1097         if err != nil {
1098             hasErr, Err := getJsonSyntaxError(requestJson)
1099             if hasErr {
1100                 err = fmt.Errorf("Invalid JSON in Incoming Request: %s", Err)
endpoints/openrtb2/auction.go
```
Malformed Json was simplified  a bit.